widgets/google-map/google-map.php

```diff
@@ -750,6 +746,15 @@ public function modify_instance( $instance ) {
 				}
 			}
 
+			// Migrate draggable and scroll_zoom to gesture_handling
+			if ( isset( $instance['settings']['draggable'] ) && ! $instance['settings']['draggable'] ) {
+				$instance['settings']['gesture_handling'] = 'none';
+			} elseif ( isset( $instance['settings']['scroll_zoom'] ) && ! $instance['settings']['scroll_zoom'] ) {
+				$instance['settings']['gesture_handling'] = 'cooperative';
+			} else {
+				$instance['settings']['gesture_handling'] = 'greedy';
+			}
```

Comment:
When migrating settings, to keep things clean we should `unset` the old settings that will no longer be used. I.e.
```
	unset( $instance['settings']['draggable'] );
	unset( $instance['settings']['scroll_zoom'] );
```